src/scss/_functions.scss

```diff
@@ -78,27 +60,30 @@
 	@return if($family, $family, '');
 }
 
-/// Path to a font asset
-///
-/// @param {String} $asset
-///
-/// @return {String} - Path to the font asset, without the file extension
-@function oFontsUseAsset($asset) {
-	@return $o-fonts-path + $asset;
-}
-
 /// Machine-readable CSS font-weight.
 ///
-/// @param {String} $keyword - Human-readable keyword, one of $_o-fonts-weights
-///
 /// @example scss
-/// font-weight: oFontsWeight(lighter);
+/// 	font-weight: oFontsWeight(lighter);
 ///
+/// @param {String} $keyword - Human-readable keyword e.g. 'semibold', 'regular', 'bold'.
 /// @return {Number} - CSS font-weight
 @function oFontsWeight($keyword) {
-	@if map-has-key($_o-fonts-weights, $keyword) {
-		@return map-get($_o-fonts-weights, $keyword);
-	} @else {
-		@warn 'Keyword "#{$keyword}" not found. Must be one of $_o-fonts-weights.';
+	@if not map-has-key($_o-fonts-weights, $keyword) {
+		@error 'Font weight "#{$keyword}" not found. Must be one of #{inspect(map-keys($_o-fonts-weights))}.';
 	}
+
+	@return map-get($_o-fonts-weights, $keyword);
+}
+
+/// Capitalise first letter of $string
+///
+/// @access private
+/// @param {String} $string - string to update
+/// @return {String} - the string with a capitalised first letter
+@function _oFontsStringCapitalise($string) {
+	$string: $string + unquote(''); // Make sure $string has a type of String
```

Comment:
once we have sass modules i'd love to introduce some common sass functions to `o-utils`, expanding its responsibility a bit and letting us put things like this and maybe math, sort in a common library

Reply:
💯 - I'd love something similar for argument validation